README.md

```diff
@@ -191,3 +192,126 @@ If you have questions, open an issue and ask us. We're looking forward to hearin
+
+1.  Open the **Command Prompt** using **Run as Administrator** in the shortcut menu.
+
+1.  [Install Chocolatey](https://chocolatey.org/install).
+
+    ```cmd
+    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+    ```
+
+1.  Verify Chocolatey was added to the environment variables:
+
+    -  In the Windows UI, open search and type `path`.
+    -  In the Windows CMD console, type `echo %path%`.
+    
+    You should see `C:\ProgramData\chocolatey\bin` in the path.
+
+1.  Close and reopen the command prompt before using `choco` commands.
+
+After running the script at the command line, you can install any required extensions. Chocolately has many extensions available, similar to Homebrew for MacOS. As a best practice, only use extensions labeled as a "trusted package". You can install editors, such as Nano and Notepad++, using Chocolatey, as well.
+
+#### Install Ruby extension
+
+If you have Ruby installed on the workstation, then you can skip this installation.
+
+1.  Open the **Command Prompt** using **Run as Administrator** in the shortcut menu.
+
+1.  Install the ruby extension:
+
+    ```cmd
+    choco install ruby
+    ```
+
+1.  Verify the environment variables were added properly:
+
+    -  In the Windows UI, open search and type `path`.
+    -  In the Windows CMD console, type `echo %path%`.
+
+>  **NOTE**  
+>  If you encounter problems with ruby, or the `gem` command is not recognized, you can install the `rubyinstaller-devkit.exe` development kit located in the `c:\ProgramData\chocolatey\bin` folder.
+
+### Install Git for Windows
+
+Use Git for Windows to prevent interference with existing Windows environment and to have Git launch commands available on the shortcut menu.
+
+Open the Git Setup file downloaded from the Git for Windows site and use the following settings during installation wizard:
+
+-  select **Use Git from Git Bash only**
+-  select **Checkout as-is, commit Unix-style line endings**
+-  select your preferred editor (can use Nano, Notepad++, or VIM)
+-  select **Enable symbolic links**
+
+Although you can install Git using Chocolatey, I chose to install _Git for Windows_ independently for more control of the installation settings.
+
+#### Set up SSH key
+
+1.  Open Git Bash. The Git Bash command is on the shortcut menu.
+
+1.  Create a working directory for Git repositories and change to the new directory.
+
+    ```bash
+    mkdir <directory-name>
+    ```
+
+1.  Follow the [Generating a new SSH](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/) instructions.
+
+
+### Connect to the DevDocs repository
```

Comment:
"Connect" is an awkward term for what's actually happening here, which is cloning, installing deps, and building locally.